### DISCUSSION TITLE
RBS Definitions

### DISCUSSION THREAD

**Discussion by cperezabo** [jeremyevans/roda#342] [General]
Hi, it would be nice to have RBS definitions so we can use tools like Steep. Any plans to add them?

**Comment by jeremyevans** [comment]
No.

**Comment by cperezabo** [comment]
No but interested or not and closed? Because maybe I could help!

**Reply by jeremyevans** [reply]
I think the cost of maintaining RBS definitions is higher than the benefits, even if someone else submitted working RBS definitions.

**Reply by HoneyryderChuck** [reply]
@cperezabo you can contribute and help maintain RBS definitions for roda and other gems [here](https://github.com/ruby/gem_rbs_collection).

**Reply by cperezabo** [reply]
Hi @HoneyryderChuck, yes, I'm aware of that, I was just looking for some official initiative before thinking on the possibility of publishing definitions there.
